lib/Doctrine/Export/Mssql.php

```diff
@@ -46,11 +46,11 @@ public function createDatabase($name)
         $query = "CREATE DATABASE $name";
         $options = $this->conn->getOptions();
         if (isset($options['database_device']) && $options['database_device']) {
-            $query.= ' ON '.$this->conn->options['database_device'];
-            $query.= $this->conn->options['database_size'] ? '=' .
-                     $this->conn->options['database_size'] : '';
```

Comment:
The `$options` property on `Doctrine_Connection` is protected, so can't access it directly. Changing these to call `getOption` instead.